server/graphql/v2/object/Host.js

```diff
@@ -437,6 +464,142 @@ export const Host = new GraphQLObjectType({
           };
         },
       },
+      contributionStats: {
+        type: new GraphQLNonNull(ContributionStats),
+        args: {
+          account: {
+            type: new GraphQLList(new GraphQLNonNull(AccountReferenceInput)),
+            description: 'A collection of accounts for which the contribution stats should be returned.',
+          },
+          dateFrom: {
+            type: GraphQLDateTime,
+            description: 'Calculate contribution statistics beginning from this date.',
+          },
+          dateTo: {
+            type: GraphQLDateTime,
+            description: 'Calculate contribution statistics until this date.',
+          },
+        },
+        async resolve(host, args, req) {
+          if (!req.remoteUser?.isAdmin(host.id)) {
+            throw new Unauthorized('You need to be logged in as an admin of the host to see the contribution stats.');
+          }
+          const where = {
+            HostCollectiveId: host.id,
+            kind: TransactionKind.CONTRIBUTION,
+            type: TransactionTypes.CREDIT,
+            isRefund: false,
+            RefundTransactionId: null,
+          };
+          const numberOfDays = getNumberOfDays(args.dateFrom, args.dateTo, host);
```

Comment:
If dateFrom & dateTo are the same day (which happens if you pick today), this will equal 0.

`contributionsAmountSum / 0` = ![](https://media.giphy.com/media/3og0IFrHkIglEOg8Ba/giphy.gif)

```suggestion
          const numberOfDays = getNumberOfDays(args.dateFrom, args.dateTo, host) || 1;
```

Reply:
Thanks for pointing it out. I've handled this both here as well as in the expense stats. 👍🏽